patches/api/0427-Add-Player-sendEquipmentChange-Map-API.patch

```diff
@@ -0,0 +1,40 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Aya <31237389+tal5@users.noreply.github.com>
+Date: Fri, 20 Jan 2023 13:49:35 +0000
+Subject: [PATCH] Add Player#sendEquipmentChange(Map) API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index de960716478477ce199526b8f860cfafa1541ee9..e98acbb96bfe226f929c3d24be24d0e5ae80e83d 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -657,6 +657,7 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+     public void sendMultiBlockChange(@NotNull java.util.Map<Location, BlockData> blockChanges, boolean suppressLightUpdates);
+     // Paper end
+ 
++    // Paper start
```

Comment:
Move this paper block down to the actual start of the diff, e.g. right above the change to a default method.